dialdbg/src/main.rs

```diff
@@ -197,10 +199,21 @@ async fn main() -> Result<()> {
             )?;
         log_config_setter = Some(log4rs::init_config(config)?);
 
-        dial_grpc(uri.as_str(), credential.as_str(), credential_type.as_str()).await;
+        let ch = dial_grpc(uri.as_str(), credential.as_str(), credential_type.as_str()).await;
```

Comment:
(nit) preference for moving this down to line 206 where it's being used (and optionally just combining the two lines `if let Some(ch) = dial_grpc(...)`).

Reply:
Good point; normally I'd agree, but I just wanted the parsed log information to appear before average RTT.

Reply:
Ahh sure, that makes sense. Okay, disregard!